main.go

```diff
@@ -22,7 +22,7 @@ var acceptFormats = []responder{
 	},
 	responder{
 		accept:      "text/plain",
-		contentType: "application/vnd.google.protobuf;; pproto=io.prometheus.client.MetricFamily; encoding=delimited",
+		contentType: "text/plain; charset=utf-8",
```

Comment:
Should we be adding the version tag in here too? the docs say when it is missing it falls back to the "latest" version, which may not be what the adapter is outputting in the future.

https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format